src/lib/app/apps.service.ts

```diff
@@ -53,6 +53,7 @@ export interface IApp {
   system_id?: string;
   containerPort?: number;
   favorite?: boolean;
+  isIconUrl?: boolean;
```

Comment:
We shouldnt add this here since its not actually part of the App contract. Lets have a small function that does this inside the component.